airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/parquet_parser.py

```diff
@@ -119,6 +119,8 @@ def _to_output_value(parquet_value: Scalar, parquet_format: ParquetFormat) -> An
                 "values": parquet_value.dictionary.tolist(),
             }
         if pa.types.is_map(parquet_value.type):
+            if not parquet_value.as_py():
+                return None
             return {k: v for k, v in parquet_value.as_py()}
```

Comment:
Can you add a unit test to make easier to understand the downstream impact of this change?